src/amqpcat.cr

```diff
@@ -86,6 +104,12 @@ class AMQPCat
     end
   end
 
+  private def write_headers(io, headers)
+    headers.each do |k, v|
+      io << "#{k}=#{v}\n"
```

Comment:
I guess this isn't performance sensitive, but I think this might be faster:
```suggestion
      io << k << "v" << k << "\n"
```
But not sure, might be some string interpolation magic which makes it just as efficent..

Reply:
updated to your suggestion